estuary/api/v1.py

```diff
@@ -276,7 +276,18 @@ def get_siblings(resource, uid):
         serialized_node['resource_type'] = inflated_node.__label__
         serialized_results.append(serialized_node)
 
-    return jsonify(serialized_results)
+    item_type = item.__label__
+    item_id = item.id_
+    header = create_header(item_type, item_id, item_story_flow, reverse)
+
+    result = {
+        'data': serialized_results,
+        'meta': {
+            'header': header
```

Comment:
could you please change the key to something more specific like `siblings_table_header`